ui-components/src/locales/general.json

```diff
@@ -1235,6 +1235,205 @@
     "WI": "Wisconsin",
     "WY": "Wyoming"
   },
+  "neighborhoods": {"AirportSub": "Airport Sub",
```

Comment:
Super nit: could you fix the indenting on this dictionary?

Reply:
oops, fixed!